colrev/ops/built_in/load_conversion/ris_rispy_loader.md

```diff
@@ -4,21 +4,23 @@ Supported File type: `ris`
 
 ## Supported fields:
 
-"first_authors"
-"secondary_authors"
-"authors"
-Converted to "authors", also used to generate ID
+
+| RIS      | Mapped to                        |
+|:---------|:---------------------------------|
+| T2       | journal                          |
+| A1,A2,A3 | author, also used to generate ID |
+| Y1,PY    | year, also used to generate ID   |
+| SP,EP    | pages (SP--EP)                   |
+| IS       | Number                           |
+| N2       | abstract                         |
+| N1       | Note                             |
+| T1,TI    | Title                            |
+
+* Check Wiki for more details about available fields, right now only the fields
+mentioned here are available.
 
 Following fields are taken just as it is:
-"title",
-"primary_title",
-"journal",
-"volume",
-"year",
-"doi",
-"publisher",
-"starting_page"
-"final_page"
+"TI", "VL", "DO", "PB", "IS"
```

Comment:
Please add the VL/DO/PB fields to the table. Instead of listing fields that are taken as is, we could explain those cases where fields are not taken as is.

Reply:
Okay